x-pack/plugins/observability/server/types/rest_specs/slo.ts

```diff
@@ -6,10 +6,22 @@
  */
 
 import * as t from 'io-ts';
-import { commonSLOSchema } from '../schema';
+import {
+  budgetingMethodSchema,
+  indicatorSchema,
+  objectiveSchema,
+  timeWindowSchema,
+} from '../schema';
 
 const createSLOParamsSchema = t.type({
-  body: commonSLOSchema,
+  body: t.type({
```

Comment:
💬 `rest_specs` now contains only the API schema as well as the application service params  (input) and response (output).